openmetadata-service/src/main/java/org/openmetadata/service/apps/AbstractNativeApplication.java

```diff
@@ -42,10 +43,10 @@
 @Slf4j
 public class AbstractNativeApplication implements NativeApplication {
   protected CollectionDAO collectionDAO;
-  private App app;
+  private @Getter App app;
   protected SearchRepository searchRepository;
-  private final CronMapper cronMapper = CronMapper.fromQuartzToUnix();
-  private final CronParser cronParser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(QUARTZ));
+  private final @Getter CronMapper cronMapper = CronMapper.fromQuartzToUnix();
```

Comment:
we can start using Record from java 17 for getter and setter

Reply:
will handle it in a followup PR. Need to understand how they work